src/audio/AudioStream.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+function AudioStream( listener ) {
+
+	this.context = listener.context;
+
+	this.gain = this.context.createGain();
+	this.gain.connect( listener.getInput() );
+
+}
+
+Object.assign( AudioStream.prototype, {
+
+	getOutput: function () {
+
+		return this.gain;
+
+	},
+
+	setAudioElement: function ( audioElement ) {
```

Comment:
To be more precise: `.createMediaElementSource()` returns an object of type `MediaElementAudioSourceNode`. I'm not 100% sure about that but this type of node is something different than `AudioBufferSourceNode`.

https://developer.mozilla.org/de/docs/Web/API/MediaElementAudioSourceNode